pandas/tests/series/test_dtypes.py

```diff
@@ -94,6 +94,14 @@ def cmp(a, b):
         result = ser.astype("object").astype(CategoricalDtype())
         tm.assert_series_equal(result, roundtrip_expected)
 
+    def test_categorical_int_to_int32(self):
```

Comment:
can you parameterize on the any_int_dtype, ideally this also works on any_int_nullable_dtype (but not sure that will just work).

Reply:
Done - created a new fixture `any_int_or_nullable_int_dtype`